README.md

```diff
@@ -1,3 +1,32 @@
+
 # pardus-image-writer
 
-a Python GTK App to burn ISO files on USB Flash sticks.
\ No newline at end of file
+A tool for writing image files to removable storage devices such as USB drives and SD cards.
+
+# Introduction
+
+pardus-image-writer is a simple and user-friendly tool for writing image files to removable storage devices. It's designed to be fast, reliable, and easy to use, making it the perfect solution for anyone who needs to write images to removable storage quickly and efficiently.
+
+# Features
+- Cross-platform compatibility: pardus-image-writer is written in Python, works on many linux distributions.
```

Comment:
Cross Platform compatibility generally means: "it works Windows, Linux & macOS"

This is "Linux" only.